build-ts/index.d.ts

```diff
@@ -4,4 +4,4 @@ import { NodeWebSocketTypeOptions } from "./lib/common.types";
 export declare class Client extends CommonClient {
     constructor(address: string, options?: IWSClientAdditionalOptions & NodeWebSocketTypeOptions, generate_request_id?: (method: string, params: object | Array<any>) => number);
 }
-export { default as Server } from "./lib/server";
+export { default as Server } from "./lib/Server";
```

Comment:
Any need for renaming this one?

Reply:
In case-sensitive file system, it will raise error because it can not find `./lib/server`